qdrant-landing/config.toml

```diff
@@ -81,6 +81,16 @@ keywords = "search engine, vector database, neural network, matching, filter, Sa
   name = "Andrey Vasnetsov"
   email = "info@qdrant.tech"
 
+[params.feedback]
```

Comment:
Based on my understanding of https://www.docsy.dev/docs/adding-content/feedback/#setup-1, I _think_ this should be:

```suggestion
[params.ui.feedback]
```

When I search Hugo discourse forums, https://discourse.gohugo.io/, I see references only to `params.ui.feedback`. If I understand the Docsy docs correctly, we can then see the results in Google Analytics.

Reply:
This is config for Hugo and Google Analytics can not know anything about it because GA works with a static site build. On the site, this code will look just like a number:

![image](https://github.com/qdrant/landing_page/assets/7085263/29969df1-1c54-46ff-a59b-4b02db4a90a2)


I didn't find any native parameters connected to feedback in Hugo docs, so I don't see a reason to introduce an additional level of nesting in this place, so I propose to leave it like this.

Reply:
Thanks so much! I'll keep you posted on how it goes. My understanding is that it can take up to 24 hours for the Yes/No "clicks" to appear in Google Analytics numbers. If I don't see anything tomorrow morning (my time -- US Pacific), I might `s/params.feedback/params.ui.feedback/` then.